Sources/Momento/config/CacheConfigurations.swift

```diff
@@ -1,20 +1,24 @@
 /// Prebuilt configurations for Momento Cache clients
-public enum CacheConfigurations {
-
-    public enum Default {
+public enum CacheClientConfigurations {
 
+    public enum iOS {
         public static func latest() -> CacheClientConfiguration {
             return CacheClientConfiguration(
                 transportStrategy: StaticTransportStrategy(
                     grpcConfig: StaticGrpcConfiguration(deadline: 15.0)
                 )
             )
         }
-
-        public static func v1()  -> CacheClientConfiguration {
-            return latest()
+    }
+    
+    public enum macOS {
+        public static func latest() -> CacheClientConfiguration {
```

Comment:
Can you go ahead and add a `default()` version for all of these these that just calls `latest()` please?

Reply:
hmm it gives me an error when trying to name it `default()`: `Keyword 'default' cannot be used as an identifier here`

Reply:
does `defaultConfig()` work as a name?

Reply:
I think I'm good with that. @cprice404 what do you think about this naming?

Reply:
Let's go with that for now. No need to block the PR on this. Please reach out to @cprice404 though at the next bug bash to verify he's ok with this naming.